native/common/jp_objecttype.cpp

```diff
@@ -82,6 +53,17 @@ JPMatch::Type JPObjectType::getJavaConversion(JPJavaFrame* frame, JPMatch& match
 		return match.type = JPMatch::_implicit;
 	}
 
+	// Apply user supplied conversions
+	if (!m_Hints.isNull())
+	{
+		JPClassHints *hints = ((PyJPClassHints*) m_Hints.get())->m_Hints;
```

Comment:
This seems ugly. Why do you need to cast and then access a private member?

Reply:
We can make an accessor.  PyJPClassHints is being stored as a PyObject because we need to hold a reference count.  But we have to get to the C++ element in order to make the conversion.  This pattern was improved in numbers2 PR.

Reply:
Thanks